dev/.env.dev.example

```diff
@@ -28,7 +28,7 @@ SOCIAL_AUTH_REDIRECT_IS_HTTPS=False
 GRAFANA_INCIDENT_STATIC_API_KEY=
 GRAFANA_API_URL=http://localhost:3000
 
-CELERY_WORKER_QUEUE="default,critical,long,slack,telegram,webhook,retry,celery"
+CELERY_WORKER_QUEUE=default,critical,long,slack,telegram,webhook,retry,celery
```

Comment:
removed quotes, as I found that starting `./celery_with_exporter.sh` locally will create queues `"default`, `celery"` with quotes.